client/patch-react.js

```diff
@@ -105,11 +106,11 @@ function wrap (fn, around) {
     return around.call(this, fn, ...args)
   }
 
-  // copy all properties
-  Object.assign(_fn, fn)
-  _fn.displayName = fn.displayName || fn.name
```

Comment:
I think we need to set the displayName manually.
If there's a displayName that's fine.

But if that's not, we need to use fn's className as the display name.